torch/onnx/symbolic_opset12.py

```diff
@@ -47,16 +47,26 @@ def outer(g, input, other):
     return einsum_helper(g, "i,j->ij", [input, other])
 
 
-@symbolic_helper.parse_args("v", "f", "i")
-def dropout(g, input, p, train):
+def _dropout_shared(g, input, p, train):
```

Comment:
I would annotate the type

Reply:
It will throw, so no exporter function does that.

Reply:
What is the error you see?

Reply:
We are working on type annotating more of our codebase. Let’s work together to see if we can do anything in this particular case

Reply:
Hopefully you can also enforce/improve documentation on new/existing components. Debugging Pytorch is not easy. I remember calling `torch.add` results 40+ functions in the call stack, and I just don't know which function went wrong...